src/app/modules/rup/components/ejecucion/puntoInicio.html

```diff
@@ -7,6 +8,7 @@
                     <div class="clearfix">
                         <div class="page-title float-left">Punto de inicio | Consultorio</div>
                         <div class="float-right">
+                            <plex-button title="Actualizar agendas" icon="mdi mdi-refresh" (click)="refreshAgenda()" [type]="'success'"></plex-button>
```

Comment:
El styling de Andes, indica que los botones tienen label o icono. No las dos cosas. 
```suggestion
                            <plex-button icon="mdi mdi-refresh" (click)="refreshAgenda()" [type]="'success'"></plex-button>
```

Reply:
No me quedo claro... esto, si es por la etiqueta title.. al poner el mouse sobre este boton aparece el tooltip seteado en tittle.

Reply:
Si, tenes razón. Interprete mal, era la etiqueta label que no va.